modules/ROOT/pages/https-reference.adoc

```diff
@@ -2,9 +2,33 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
+:keywords: configure, HTTPS endpoint
+
+You can automatically deploy an API proxy using an HTTP listener (inbound HTTP endpoint) with minimal setup. You can use this approach if you do not foresee any security concerns in your Anypoint Platform API setup. Otherwise, you must configure your API proxy with an HTTPs listener (inbound HTTPS endpoint), instead of an HTTP listener.
+
+MuleSoft requires that you provide your own certificate when configuring your HTTPs setup. The certificate you use can be a self-signed certificate to be used for demo purposes, custom TLS certificate with validity within your organization, or a trusted Certificate Authority (CA) certificate.
```

Comment:
Consider this reformatting to emphasize the list:

```suggestion
MuleSoft requires that you provide your own certificate when configuring your HTTPS setup. Use one of the following types:
* A self-signed certificate used for demo purposes
* A custom TLS certificate with validity within your organization
* A trusted Certificate Authority (CA) certificate
```

Reply:
Done! Thanks!